README.md

```diff
@@ -20,6 +20,8 @@ Other than tracing, Tracee is also capable of capturing files written to disk or
 To run, Tracee requires the following:
 - Linux kernel version > 4.14
 - Kernel headers
+
+In addition, if running the binary directly (not via docker):
```

Comment:
I don't think this is needed.. libc and clang are always required. in docker we baked them in the image, but they are still required by tracee.

Reply:
ok